### PR TITLE
[3.2] Reduce producer_plugin debug level log spam

### LINKED ISSUE #1003: 4.0.0-rc3: usless debug message during catch up "Subjective billed accounts 0 removed 0"
When starting from a snapshot and catching up blocks to get to current, you get a LOT of debug messages like this:

```
debug 2023-04-11T13:35:07.865 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
```

These messages seem unhelpful in "catch-up mode" and could be suppressed.

Eventually you get other messages and it is fine.

```
debug 2023-04-11T13:35:08.277 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
debug 2023-04-11T13:35:08.278 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
debug 2023-04-11T13:35:08.279 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
debug 2023-04-11T13:35:08.280 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
debug 2023-04-11T13:35:08.281 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
debug 2023-04-11T13:35:08.282 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
debug 2023-04-11T13:35:08.282 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
debug 2023-04-11T13:35:08.283 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
debug 2023-04-11T13:35:08.283 nodeos    producer_plugin.cpp:587       on_incoming_block    ] received incoming block 70967643 043ae15bfc53fcfcd33c0c9c89da1cf8e917cc01e3d16c87a0126a2aea1811c0
debug 2023-04-11T13:35:08.284 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
info  2023-04-11T13:35:08.284 nodeos    producer_plugin.cpp:651       on_incoming_block    ] Received block fc53fcfcd33c0c9c... #70967643 @ 2023-04-11T13:30:08.500 signed by ohtigertiger [trxs: 1, lib: 70967313, confirmed: 0, net: 184, cpu: 537, elapsed: 398, time: 558, latency: 299784 ms]
debug 2023-04-11T13:35:08.284 nodeos    producer_plugin.cpp:587       on_incoming_block    ] received incoming block 70967644 043ae15c4bd895d164b13778158467b3724909a157d6738ae4463b9fa8c87e40
debug 2023-04-11T13:35:08.285 nodeos    subjective_billing.hpp:160    on_block             ] Subjective billed accounts 0 removed 0
info  2023-04-11T13:35:08.285 nodeos    producer_plugin.cpp:651       on_incoming_block    ] Received block 4bd895d164b13778... #70967644 @ 2023-04-11T13:30:09.000 signed by ohtigertiger [trxs: 3, lib: 70967313, confirmed: 0, net: 552, cpu: 1389, elapsed: 908, time: 1105, latency: 299285 ms]
debug 2023-04-11T13:35:08.285 nodeos    producer_plugin.cpp:587       on_incoming_block    ] received incoming block 70967645 043ae15db5f961663ba91883a7abf6481af3ada8ca7d36b99001d0f14736a004
```

### PR DESCRIPTION
**Pull request by heifner**
Reduce `producer_plugin` `debug` level log spam.

Resolves #1003

### CODE REVIEW

File: plugins/producer_plugin/producer_plugin.cpp [hunk 1]
```diff
@@ -461,9 +463,11 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
          const auto& id = block_id ? *block_id : block->calculate_id();
          auto blk_num = block->block_num();
 
-         fc_dlog(_log, "received incoming block ${n} ${id}", ("n", blk_num)("id", id));
+         auto now = fc::time_point::now();
+         if (now - block->timestamp < fc::minutes(5) || (blk_num % 1000 == 0)) // only log every 1000 during sync
```
[the remark this answers is not in the document]
Comment (heifner) [reply plugins/producer_plugin/producer_plugin.cpp:467]: https://github.com/AntelopeIO/leap/pull/805#discussion_r1133115044 :)

[the remark this answers is not in the document]
Comment (linh2931) [reply plugins/producer_plugin/producer_plugin.cpp:467]: I thought I saw this somewhere, but did not find it. Thanks.

**Review by linh2931 — APPROVED** [review]

File: plugins/producer_plugin/producer_plugin.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (greg7mdp) [reply plugins/producer_plugin/producer_plugin.cpp:467]: Still, should be `&&` I think, not `||`.

[the remark this answers is not in the document]
Comment (heifner) [reply plugins/producer_plugin/producer_plugin.cpp:467]: We want every block logged when synced up to current.

[the remark this answers is not in the document]
Comment (greg7mdp) [reply plugins/producer_plugin/producer_plugin.cpp:467]: won't `now - block->timestamp < fc::minutes(5)`  be true while syncing?

**Review by greg7mdp — APPROVED** [review]

**Pull request merged**